cirq-core/cirq/sim/act_on_density_matrix_args.py

```diff
@@ -35,7 +35,9 @@ class ActOnDensityMatrixArgs(ActOnArgs):
 
     def __init__(
         self,
-        target_tensor: np.ndarray,
+        target_tensor: Optional[np.ndarray] = None,
+        initial_state: Union[np.ndarray, 'cirq.STATE_VECTOR_LIKE'] = 0,
+        dtype: Type[np.number] = np.complex64,
```

Comment:
Any new parameters need to be last in the list, since users may be calling the constructor without kwargs.

Reply:
@yjt98765 This still needs moved to the bottom.

Reply:
Sorry, I missed it.

Reply:
Thanks, you can resolve this one.